### PR TITLE
sua loi da ngon ngu

### PR DESCRIPTION
**Pull request by nhkhanh68**

### CODE REVIEW

File: app/views/static_pages/home.html.erb [hunk 1]
```diff
@@ -7,8 +7,13 @@
     sample application.
   </h2>
 
-  <%= link_to "Sign up now!", signup_path, class: "btn btn-lg btn-primary" %>
+  <%= link_to t("signup"), signup_path, class: "btn btn-lg btn-primary" %>
+  	<p>
```
Comment (lucifer0811) [inline app/views/static_pages/home.html.erb]: indent

File: app/controllers/static_pages_controller.rb [hunk 2]
```diff
@@ -1,7 +1,8 @@
 class StaticPagesController < ApplicationController
   def home
+  	flash[:notice] = t(:hello_flash)
```
Comment (halo93) [inline app/controllers/static_pages_controller.rb]: Remove () + Indent

File: app/views/static_pages/home.html.erb (hunk 1 above)
Comment (halo93) [inline app/views/static_pages/home.html.erb]: Indent

File: app/views/static_pages/home.html.erb [hunk 3]
```diff
@@ -7,8 +7,13 @@
...
+	<%= link_to_unless I18n.locale == :vn, "VietNam", locale: :vn %>
+	</p>
+  	<h1><%=t :hello_world %></h1>
+	<h1><%= flash[:notice] %></h1>
 </div>
-
-<%= link_to image_tag("rails.png", alt: "Rails logo"),
-            'http://rubyonrails.org/' %>
\ No newline at end of file
+<%= link_to image_tag("rails.png", alt: "Rails logo"),'http://rubyonrails.org/' %>
```
Comment (lucifer0811) [inline app/views/static_pages/home.html.erb]: double quote
1 khoảng trắng

File: config/locales/en.yml [hunk 4]
```diff
@@ -21,3 +21,6 @@
 
 en:
   hello: "Hello world"
+  hello_world: Hello world!
+  hello_flash: Hello flash!
+  signup     : Sign Up!
```
Comment (halo93) [inline config/locales/en.yml]: signup: Sign up!

File: config/locales/vn.yml [hunk 5]
```diff
@@ -0,0 +1,4 @@
+vn:
+  hello_world: Xin chao!
+  hello_flash: Xin chao flash!
+  signup:      Dang ky!
```
Comment (halo93) [inline config/locales/vn.yml]: signup: Đăng ký!

File: app/views/static_pages/home.html.erb [hunk 6]
```diff
@@ -1,14 +1,17 @@
...
   <h2>
     This is the home page for the
     <a href="http://www.railstutorial.org/">Ruby on Rails Tutorial</a>
     sample application.
   </h2>
-
-  <%= link_to "Sign up now!", signup_path, class: "btn btn-lg btn-primary" %>
+  <%= link_to t("signup"), signup_path, class: "btn btn-lg btn-primary" %>
+  <p>
+	<%= link_to_unless I18n.locale == :en, "English", locale: :en %>
```
Comment (lucifer0811) [inline app/views/static_pages/home.html.erb]: indent

File: app/controllers/static_pages_controller.rb [hunk 7]
```diff
@@ -1,5 +1,6 @@
 class StaticPagesController < ApplicationController
   def home
+  	flash[:notice] = t :hello_flash
```
Comment (lucifer0811) [inline app/controllers/static_pages_controller.rb]: indent

**Pull request merged**